packages/dev/docs/pages/blog/rtl-date-time.mdx

```diff
@@ -0,0 +1,117 @@
+  <span>:</span>
+  <span aria-label="minute">45</span>
+  <span>{'\u2069'}</span>
+</div>
+```
+
+## DateFields
+
+Date fields, on the other hand, were much more complicated to solve in comparsion. Since the Unicode Bidirectional algorithm was not formatting the segments, the resulting format appeared to mirror the order in which the segments were stored, as returned by [DateFormatter](https://react-spectrum.adobe.com/internationalized/date/DateFormatter.html#dateformatter). This suggested that we could apply a similar approach to what we used for time fields — forcing a left-to-right direction on the date segments. However, this assumption proved too broad. In some locales, such as `ar-AE`, the date segments were already correctly formatted, meaning that enforcing a left-to-right direction would make it incorrect. We found that in Arabic, the separators between date segments contained [right-to-left marks](https://en.wikipedia.org/wiki/Implicit_directional_marks) which was the reasoning behind why the date segments were already correctly formatted. In contrast, Hebrew did not have such markers. Therefore, we had to adopt a different approach that accounted for these variations. 
+
+
+```tsx example render=false
+// An example of a date field in ar-AE
+<div>
+  <span aria-label="day">3</span>
+  <span>&rlm;/</span>
+  <span aria-label="month">11</span>
+  <span>&rlm;/</span>  
+  <span aria-label="year">2020</span>
+</div>
+```
+
+```tsx example render=false
+// An example of a date field in he-IL
+<div>
+  <span aria-label="day">3</span>
+  <span>.</span>
+  <span aria-label="month">11</span>
+  <span>.</span>  
+  <span aria-label="year">2020</span>
+</div>
+```
+
+Through much trial and error, we discovered that appplying the [left-to-right embedding (LRE) Unicode](https://unicode.org/reports/tr9/#Explicit_Directional_Embeddings) on the date segments allowed us to to treat the text as embedded left-to-right while preserving the right-to-left mark on the separators, ensuring that Arabic dates display in the correct format. While we could have added Unicode to the segments like we did with the time fields, we opted for the [equivalent CSS](https://unicode.org/reports/tr9/#Markup_And_Formatting) approach instead to avoid modifying the DOM. This CSS was applied on date segments with placeholder or actual values to avoid the behavior discussed earlier with shifting segments. Through additional testing, we found that we should only apply left-to-right embedding on numeric values. If the value was displayed as text, say "November" instead of "11", we did not apply this CSS. 
+
+## Keyboard Navigation
+
+After fixing the formatting though, we also needed to update the keyboard navigation. Previously, if pressing the left arrow key, you would go to the next node in the DOM and vice versa for the right arrow key. After these changes though, visually adjacent elements were not necessarily adjacent in the DOM so this would not work anymore. 
```

Comment:
```suggestion
After fixing the formatting, we also needed to update the keyboard navigation. Previously, when pressing the left arrow key, you would go to the next node in the DOM and vice versa for the right arrow key. After these changes though, visually adjacent elements were not necessarily adjacent in the DOM so this would not work anymore. 
```